pkg/thing/interactors/update_config.go

```diff
@@ -39,68 +39,78 @@ func (i *ThingInteractor) validateConfig(authorization, id string, configList []
 		return fmt.Errorf("error getting thing metadata: %w", err)
 	}
 
-	if thing.Schema == nil {
-		return ErrSchemaUndefined
+	err = validateSchemaExists(configList)
+	if err != nil {
+		return err
 	}
```

Comment:
I think here should a more complete schema validation, just like was in `updateSchema`.

Reply:
Done!